web/package.json

```diff
@@ -41,7 +41,10 @@
     "route-snapper-ts": "^0.0.8",
     "svelte-maplibre": "^0.9.14",
     "svelte-toc": "^0.5.9",
-    "svelte-utils": "github:a-b-street/svelte-utils",
+    "svelte-utils": "github:a-b-street/svelte-utils#6a5b755ae7f8216c5b8996333b16221f1b344595",
```

Comment:
We need to pin a few things, because in the latest commit in these packages, I've switched to Svelte 5